src/Components/VersionTable/VersionTable.js

```diff
@@ -0,0 +1,145 @@
+
+import "./VersionTable.scss";
+
+const VersionTable = () => {
+  const { t } = useTranslation();
+
+  const serverInfo = useSelector((state) => state.login.serverInfo);
+  const [desktopInfo, setDesktopInfo] = useState(null);
+
+  useEffect(() => {
+    if (window.VOCASCAN_CONFIG.ENV === "electron") {
+      getDesktopVersions().then((versions) => {
+        setDesktopInfo(versions);
+      });
+    }
+  }, []);
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: t("components.versionTable.headers.component"),
+        accessor: "component",
+        Cell: ({ row, value }) => (
+          <a
+            href={`https://github.com/vocascan/${row.original.component}`}
+            rel="noreferrer"
+            target="_blank"
+          >
+            {value}
+          </a>
+        ),
+      },
+      {
+        Header: t("components.versionTable.headers.version"),
+        accessor: ({ version }) => `v${version}`,
+      },
+      {
+        Header: t("components.versionTable.headers.commit"),
+        accessor: "commit",
+        Cell: ({ row, value }) =>
+          value ? (
+            <a
+              href={`https://github.com/vocascan/${row.original.component}/tree/${value}`}
+              rel="noreferrer"
+              target="_blank"
+            >
+              #{value.slice(0, 7)}
+            </a>
+          ) : (
+            ""
+          ),
+      },
+      {
+        Header: t("components.versionTable.headers.build"),
+        accessor: "",
+        Cell: ({ row, value }) =>
+          row.original.runId && row.original.runNumber ? (
+            <a
+              href={`https://github.com/vocascan/${row.original.component}/actions/runs/${row.original.runId}`}
+              rel="noreferrer"
+              target="_blank"
+            >
+              #{row.original.runId}-{row.original.runNumber}
+            </a>
+          ) : (
+            ""
+          ),
+      },
+      {
+        Header: t("components.versionTable.headers.date"),
+        accessor: ({ date }) =>
+          date
+            ? `${date}\n(${t("global.daysAgo", {
+                count: dayDateDiff(new Date(date), new Date()),
+              })})`
+            : "",
+        Cell: ({ value }) => <p className="table-cell">{value}</p>,
+      },
+      {
+        Header: t("components.versionTable.headers.extra"),
+        accessor: ({ extra }) =>
+          extra
+            ? Object.entries(extra)
+                .map(([key, value]) => `${key}: ${value}`)
+                .join("\n")
+            : "",
+        Cell: ({ value }) => <p className="table-cell">{value}</p>,
+      },
+    ],
+    [t]
+  );
+
+  const data = useMemo(
+    () => [
+      {
+        component: "vocascan-frontend",
+        version: webVersion,
+        ...webInfo,
+      },
+      ...(window.VOCASCAN_CONFIG.ENV === "electron"
+        ? [
+            {
+              component: "vocascan-desktop",
+              version: "1.1.0",
```

Comment:
Can't you pull that value from the package.json?

Reply:
Sorry, that was some testing code. I'll definitely fix that.